recipes/pygraphviz/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pygraphviz" %}
+{% set version = "1.4rc1" %}
```

Comment:
Ideally we would not use an rc, but they have not made a release in a *long* time. FWIW pinged issue ( https://github.com/pygraphviz/pygraphviz/issues/98 ) to see when 1.4 final might arrive.

Reply:
Yeah, 1.3 is 2 years old! Do you want me to downgrade?

Reply:
No. It's just not practical given how old it is.

More trying to provide context on why using an rc should be ok here.